lv_conf_template.h

```diff
@@ -144,6 +144,20 @@
     #endif
 #endif
 
+/*Use TSi's aka (Think Silicon) NemaGFX */
+#define LV_USE_NEMA_GFX 0
+
+#if LV_USE_NEMA_GFX
+    /*Enable Vector Graphics Operations. Available only if NemaVG library is present*/
+    #define LV_USE_NEMA_VG 0
+
+    #if LV_USE_NEMA_VG == 1
+        /*Define application's resolution used for VG related buffer allocation */
+        #define LV_NEMA_GFX_RESX 800
```

Comment:
Could this resolution be changed on the fly?

Reply:
No, this resolution is referred to internal back buffers used to do VG calculations and are always the same size in dimensions with the application RESX, RESY